docroot/themes/contrib/civictheme/templates/content/node.html.twig

```diff
@@ -0,0 +1,61 @@
+{#
```

Comment:
@joshua-salsadigital 
could you please provide more information about why this was added and what was overridden specifically in this tiwg file.

Reply:
When testing the drupal elements created a new content type. 
It goes edge to edge and no styles applied
This general template file handles an new content type with basic civic theme styles

Reply:
as discussed - please remove this change. thank you